common/beerocks/scripts/beerocks_watchdog.sh.in

```diff
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/bin/bash
```

Comment:
I suspect this change isn't possible, but it has to be checked with the owners of the platform it's used on (UGW and RDKB?).

Although the features you mentioned are not POSIX (`++`, `let`, `==`), they are available in some versions of busybox, so it's possible that the target platform doesn't have bash at all.

I just noticed it also get installed for the prplWrt targets. It's unused at the moment, but it will also become unusable since it doesn't have bash (although removing it from the prplwrt builds is also a possibility).

Reply:
Issue here, i'm not sure how to validate this script after changes required to make in POSIX complaint. That why bash-isms weren't removed.

Reply:
Better use exceptions for them then.

Reply:
This is used in RDKB and UGW, and although not used in RAX40 it is installed there. In all three platforms we don't have bash - so this can't be merged without testing.
The script is monitoring beerocks processes to bring it up if something went down, and also detect slow down of beerocks processes.

To verify the script still works after these changes (minus the shabeng ofcourse), you should run the script manually on RAX40, then kill the main agent or the controller (don't kill the son slaves cause they have internal watchdog mechanism). The watchdog script should restart the killed process.